### ISSUE TITLE
Aedict not compatible with Android 2.3.x

### ISSUE THREAD

**Issue by mvysny** [mvysny/aedict#712]
```
java.lang.NoSuchMethodError: android.view.ContextThemeWrapper.getThemeResId
	at org.jetbrains.anko.internals.AnkoInternals.wrapContextIfNeeded(Internals.kt:58)
	at sk.baka.aedict3.unitedkanjisearch.KanjiDrawFragment.onCreateView(KanjiDrawFragment.kt:190)
	at android.support.v4.app.Fragment.performCreateView(Fragment.java:1962)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1248)
	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:738)
	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1613)
	at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:330)
	at android.support.v4.app.FragmentActivity.onPostResume(FragmentActivity.java:452)
	at android.support.v7.app.AppCompatActivity.onPostResume(AppCompatActivity.java:141)
	at android.app.Activity.performResume(Activity.java:3867)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2114)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2139)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1672)
	at android.app.ActivityThread.access$1500(ActivityThread.java:117)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:935)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:130)
	at android.app.ActivityThread.main(ActivityThread.java:3691)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:507)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:912)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:670)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by mvysny** [comment]
Reported upstream: https://github.com/Kotlin/anko/issues/236

**Comment by mvysny** [comment]
Yet, Google is also dropping support for Androids older than 4.0.4, so maybe it is time to move on. Apparently, the ContextThemeWrapper.getThemeResId function is present on every Android 15 and upwards. I will thus mark Aedict to require Android 4.0.4 at least.

**Comment by mvysny** [comment]
Fixed in 3.39.27
